resources/views/project/ctest-configuration.blade.php

```diff
@@ -9,7 +9,13 @@
 set(CTEST_PROJECT_NAME {{ $project->Name }})
 set(CTEST_NIGHTLY_START_TIME {{ $project->NightlyTime }})
 
-set(CTEST_SUBMIT_URL {{ env('APP_URL') }}/submit.php?project={{ urlencode($project->Name) }})
+if(CMAKE_VERSION VERSION_GREATER 3.14)
+  set(CTEST_SUBMIT_URL {{ env('APP_URL') }}/submit.php?project={{ urlencode($project->Name) }})
+else()
+  set(CTEST_DROP_METHOD "{{ explode("://", config('app.url'))[0] }}")
+  set(CTEST_DROP_SITE "{{ explode("://", config('app.url'))[1] }}")
+  set(CTEST_DROP_LOCATION "{{ substr($_SERVER['REQUEST_URI'], 0, strpos($_SERVER['REQUEST_URI'], '/')) }}/submit.php?project={{ urlencode($project->Name) }}")
```

Comment:
We shouldn't be using PHP global variables like `$_SERVER` unless absolutely necessary.  `$request->path()` (either pass in a request object or use the request facade) would be a better alternative.  See the docs [here](https://laravel.com/docs/9.x/requests) for all of the available options.

That said, why do we need to parse the path from the request?  Can't we get that from the base URL?  Additionally, can't you just use `explode()` here rather than the current `substr()`-based approach?   If you feel like using Laravel, you could also use [`Str::before()`](https://laravel.com/docs/9.x/helpers#method-str-before).

Reply:
@williamjallen, I followed what we used to do for the paths which, I think. parsed out if CDash was being hosted at a non-root path.  Since the docker container hosts it at the root, I wasn't sure if we needed that information again or if that would be captured in APP_URL.  I'll remove the definitely wrong things and see what else I can do to test things